tools/terraform_sync_tool/terraform_sync.py

```diff
@@ -0,0 +1,60 @@
+import json
```

Comment:
Make this python tool more re-usable:

- Accept user-provided command line arguments (use argparse)
- Add main method with if __name__ == '__main__': main() at bottom of file

Reply:
Accept user-provided arguments for json file - https://github.com/candicehou07/bigquery-utils/commit/5c36be7f3696d8f9fcd6b7b8e4a5f8e0ec32b7c2

Refactor terraform_sync.py - https://github.com/candicehou07/bigquery-utils/commit/7a33b29b35ab84f5dd6189b0522a72a0e4e49858